src/libs/styles.js

```diff
@@ -1,11 +1,24 @@
-export default function addStyles(styles) {
+function getAddStyles() {
+  const memo = {};
   const id = 'atomic-fuel-styles';
+
   let styleEl = document.getElementById(id);
-  if (!styleEl) {
-    styleEl = document.createElement('style');
-    styleEl.id = id;
-    document.head.appendChild(styleEl);
-  }
-  const styleSheet = styleEl.sheet;
-  styleSheet.insertRule(styles, styleSheet.cssRules.length);
+
+  return (styles) => {
+    if (!styleEl) {
+      styleEl = document.createElement('style');
+      styleEl.id = id;
+      document.head.appendChild(styleEl);
+    }
+
+    const classes = styles.match(/.*[^{]/)[0];
```

Comment:
What does this regex do? It would be wroth adding a comment.

Reply:
I added documentation as suggested via Slack.